### PR TITLE
docs: release 1.6.0 changelog

### PR DESCRIPTION
**Pull request by changyuwong**

### CODE REVIEW

File: content/changelog/bytebase-1-6-0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- Thanks to @[fatelei](https://github.com/fatelei) for [#2965 feat(sql-review): implement postgresql check the number of index columns](https://github.com/bytebase/bytebase/pull/2965)
+- Thanks to @[zeroslope](https://github.com/zeroslope) for [#2929 feat(advisor): limit value list len in PostgreSQL INSERT statement](https://github.com/bytebase/bytebase/pull/2929)
+- Thanks to @[praneetloke](https://github.com/praneetloke) for [#2898 chore: improve error message when DB connection fails](https://github.com/bytebase/bytebase/pull/2898)
+- Thanks to @[Kikkon](https://github.com/Kikkon) for [#2874 feat(sql-review): limit PostgreSQL comment length](https://github.com/bytebase/bytebase/pull/2874)
+- Thanks to @[hamidzr](https://github.com/hamidzr) for [#2857 docs: fix minor typos](https://github.com/bytebase/bytebase/pull/2857)
+
+## 📕 Installation and Upgrade
+
+For a fresh installation, follow https://github.com/bytebase/bytebase#installation.
```
Comment (milasuperstar) [inline content/changelog/bytebase-1-6-0.md:33]:
```suggestion
Follow https://github.com/bytebase/bytebase#installation. If you upgrade from a previous version, just restart after obtaining the new release binary.
```

**Review by milasuperstar — CHANGES_REQUESTED** [review]

**Review by boojack — APPROVED** [review]
LGTM for `VERSION` file changes.

**Review by milasuperstar — APPROVED** [review]

File: content/changelog/bytebase-1-6-0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Bytebase 1.6.0
+author: Cy
```
Comment (adela-bytebase) [inline content/changelog/bytebase-1-6-0.md:3]: should be Changyu instead of Cy, or there will be no avatar

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/changelog/bytebase-1-6-0.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Bytebase 1.6.0
+author: changyu
```
Comment (boojack) [inline content/changelog/bytebase-1-6-0.md:3]:
```suggestion
author: Changyu
```

**Pull request merged**